### PR TITLE
Adding 2 new actions: prevent tab close and simple shortcut

### PR DESCRIPTION
**Pull request by dkzlv**
PR for issue #10

### CODE REVIEW

**Comment by sw-yx** [comment]
leavnig this open to invite comments, api discussion, docs notes, from anyone

**Comment by dkzlv** [comment]
Actually I found a bug in the binary logic there. If you introduce a shortcut like `shift+1` the action will fire even if you press `shift+alt+cmd+1`.

I'll try to find a simple solution to this.

**Comment by dkzlv** [comment]
@sw-yx Fixed. Saved the new demo in REPL.

**Comment by sw-yx** [comment]
thank you!

File: src/index.ts [hunk 1]
```diff
@@ -161,4 +161,76 @@ export function lazyload(node: HTMLElement, attributes: Object): ReturnType<Acti
 			observer.unobserve(node);
 		}
 	};
-}
\ No newline at end of file
+}
+
+/**
+ * Prevent current tab from beind closed by user
```
Comment (sw-yx) [inline src/index.ts:167]:
```suggestion
 * Prevent current tab from being closed by user
```

File: src/index.ts [hunk 2]
```diff
@@ -161,4 +161,75 @@ export function lazyload(node: HTMLElement, attributes: Object): ReturnType<Acti
...
+ */
+export const preventTabClose: Action = (_, enabled: boolean) => {
+  const handler = (e: BeforeUnloadEvent) => {
+      e.preventDefault();
+      e.returnValue = '';
+    },
+    setHandler = (shouldWork: boolean) =>
+      shouldWork ?
+        window.addEventListener('beforeunload', handler) :
+        window.removeEventListener('beforeunload', handler);
```
Comment (sw-yx) [inline src/index.ts:179]: TIL about https://developers.google.com/web/updates/2018/07/page-lifecycle-api#the-unload-event looks like you implemented it in the correct way

**Comment by dkzlv** [comment]
@sw-yx Wassup? :)

I wanted to submit another PR but I don't know if it makes sense if the repo is frozen.

**Comment by sw-yx** [comment]
haha hey :) not much, just wanted to invite others to comment. but i can merge :)

**Comment by sw-yx** [comment]
released as v0.1.0 https://github.com/sw-yx/svelte-actions/releases/tag/v0.1.0

**Pull request merged**